server/fleet/windows_mdm.go

```diff
@@ -29,6 +35,48 @@ type MDMWindowsConfigProfile struct {
 	UpdatedAt   time.Time `db:"updated_at" json:"updated_at"`
 }
 
+// ValidateUserProvided ensures that the SyncML content in the profile is valid
+// for Windows.
+//
+// It checks that all top-level elements are <Replace> and none of the <LocURI>
+// elements within <Target> are reserved URIs.
+//
+// Returns an error if these conditions are not met.
+func (m *MDMWindowsConfigProfile) ValidateUserProvided() error {
+	if mdm.GetRawProfilePlatform(m.SyncML) != "windows" {
+		return errors.New("Only <Replace> supported as a top level element. Make sure you don’t have other top level elements.")
+	}
+
+	doc := etree.NewDocument()
+	if err := doc.ReadFromBytes(m.SyncML); err != nil {
+		return err
```

Comment:
I think this one should be "Couldn’t upload. The file should include valid XML." (i.e. this case in the figma: https://www.figma.com/file/eC1C8kp9grdPMAZGvR0ibT/%2313281-Enforce-configuration-on-Windows?type=design&node-id=203-2196&mode=design&t=3t2OUFU7rugc41kL-0), though maybe wrapped so the internal error is kept and logged...